api/src/test/java/org/pmiops/workbench/api/DataSetControllerTest.java

```diff
@@ -960,6 +968,20 @@ public void exportToNewNotebook() {
         .saveNotebook(eq(WORKSPACE_BUCKET_NAME), eq(notebookName), any(JSONObject.class));
   }
 
+  @Test(expected = ForbiddenException.class)
```

Comment:
I just learned that there's an error code `402 Payment Required`, but it's nonstandard. Would've fit like a glove here though. https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/402